pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/service/actvityinstancebuilder/form/block/question/ValidationRuleCreator.java

```diff
@@ -207,6 +208,12 @@ private UniqueValueRule createUniqueValueRule(AIBuilderContext ctx, UniqueValueR
     }
 
     private ComparisonRule createComparisonRule(AIBuilderContext ctx, ComparisonRuleDef ruleDef) {
+        QuestionDto questionDto = TransactionWrapper.withTxn(handle -> handle.attach(QuestionDao.class).getJdbiQuestion()
```

Comment:
not checking if user has instance of the activity with this question !!

Reply:
It doesn't really matter since in `ComparisonRule.validate` there is a check if the answer is present.